partner_daily_notes/models/calendar.py

```diff
@@ -51,15 +55,15 @@ def generate_cancel_daily_note(self, cancel_reason, appointment):
                 "Cancelled %s: %s, %s %s. Reason: %s"
                 % (
                     self.type_id.name,
-                    self.start,
+                    current_time,
```

Comment:
OK.
But this whole if else should be refactored in to a normal if else instead to make it a bit easier to read.